benchmark.go

```diff
@@ -65,11 +65,18 @@ type Summary struct {
 	RPS                float64 `json:"rps"`
 }
 
+// StreamSummary stores the stream messages send and received summary
+type StreamSummary struct {
+	TotalStreamMessagesSent     int `json:"totalStreamMessagesSent,omitempty"`
+	TotalStreamMessagesReceived int `json:"totalStreamMessagesReceived,omitempty"`
```

Comment:
Why do we have omitempty for `TotalStreamMessagesSent` and `TotalStreamMessagesReceived`? when `StreamSummary` I do not see why both variables will not be used.

Reply:
There are few cases when client or server might not send/receive any message, more like a one way communication. In such cases `TotalStreamMessagesSent` or `TotalStreamMessagesReceived` could be zero. 

Example: Client sends multiple messages and closes send stream, server processes them and can choose to end the stream without response (this is a valid scenario)

```
yab --max-duration=1s --timeout=1m -F testproto --service "test-server" --method uber.yarpc.encoding.protobuf.Test/ClientStream -p grpc://localhost:55555   -r '{}{}' --format=JSON
{}
{}

{
  "benchmarkParameters": {
    "cpus": 12,
    "connections": 24,
    "concurrency": 1,
    "maxRequests": 0,
    "maxDuration": "1s",
    "maxRPS": 0
  },
  "latencies": {
    "0.5000": "649.711µs",
    "0.9000": "1.066174ms",
    "0.9500": "1.661274ms",
    "0.9900": "16.081985ms",
    "0.9990": "23.005123ms",
    "0.9995": "24.909958ms",
    "1.0000": "26.037134ms"
  },
  "summary": {
    "elapsedTimeSeconds": 1,
    "totalRequests": 25088,
    "rps": 25077.35
  },
  "streamSummary": {
    "totalStreamMessagesSent": 50176
  }
}
```

IMO zero value fields such as send and received can be printed as well, wdyt?